content/blog/2019/06/2019-06-25-Phase1-Updates-On-Working-Hours-Plugin

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "GSOC Phase 1 Updates On Working Hours Plugin"
+tags:
+- react
+- plugins
+- gsoc
+- gsoc2019
+author: shenjack
+---
+
+= UI Improvements with React on Working Hours Plugin
+
+ The Working Hour Plugin provides an interface to set up a schedule of allowable build times.  Jobs that run outside of configured working hours are held until the next allowable build time.  
+
+For the first code phase at Google Summer of Code, I've been working on link:/projects/gsoc/2019/working-hours-improvements/[Working Hours Project], which needed improvements on usability.
+
+Rather than classical Jelly pages, React seems to be more prefered when we want to design a much customized UI. But we have to face a challenge of the integration of React and Jenkins, which I'm currently working on.
```

Comment:
I think I'd here that one good reason to use React is to be able to take advantage of open source components such as date pickers